### DISCUSSION TITLE
GUI: Inference and Training windows print 'behind' the main GUI

### DISCUSSION THREAD

**Discussion by olinesn** [talmolab/sleap#995] [Ideas]
Small usability item - on Windows 10 and 11, after selecting Inference or Training from the "Predict" drop-down menu, the pop-up menus strangely print behind the main GUI. This has the effect of being unclear to the user that the program has responded to the user's request. It can be especially inconvenient if the menu takes a while to pop up - then you're repeatedly checking the taskbar or listening for the OS beep.

Solution would be to have the Training/Inference menus print on top of the main SLEAP GUI.

**Comment by gitttt-1234** [comment]
Thanks for reporting this. This was caused by the training/inference dialogs being created without a parent widget on Windows.

We've fixed this in PR #2703 — the dialogs now correctly reference the main window as their parent, which ensures they appear on top as expected.

Closing this discussion!
